torch/_functorch/_aot_autograd/jit_compile_runtime_wrappers.py

```diff
@@ -179,11 +185,25 @@ def aot_dispatch_base(
     compiled_fw = functionalized_rng_wrapper.post_compile(
         compiled_fw, aot_config, runtime_metadata=fw_metadata
     )
+    if config.enable_autograd_cache and aot_config.cache_key:
+        if fw_key := getattr(compiled_fw, "_fx_graph_cache_key", None):
```

Comment:
(dupe) when can this be None?

Reply:
If the bw_compiler is not inductor or FXGraphCache is disabled, then the returned CompiledFunction won't have a cache key associated with it. Basically, AOTAutogradCache will only save a cache entry when FXGraphCache successfully saves one.